stl/inc/optional

```diff
@@ -104,7 +104,7 @@ struct _Optional_destruct_base<_Ty, false> { // either contains a value of _Ty o
 
     _CONSTEXPR20 ~_Optional_destruct_base() noexcept {
         if (_Has_value) {
-            _Destroy_in_place(_Value);
+            _Value.~_Ty();

```

Comment:
No change requested: I observe that we're storing `remove_cv_t<_Ty> _Value;` and saying only `~_Ty()`. This seems to be acceptable according to the Standardese, and all compilers accept, so I don't think we need to add `remove_cv_t` here.